apps/frontend/src/libs/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import styles from "./styles.module.css";
+
+type Properties = {
+	name?: string;
```

Comment:
We should only render header in a place where we have user
```suggestion
	name: string;
```